aws/utils.go

```diff
@@ -225,6 +226,11 @@ func handleEmptySliceAndMap(ctx context.Context, d *transform.TransformData) (an
 		if reflectVal == reflect.Zero(reflectVal.Type()) {
 			return nil, nil
 		}
+	case reflect.String:
+		// To handle empty string to null change in aws sdk V1 to V2 migration
```

Comment:
@LalitTurbot Do we still need this since we're not doing v1 backward compatible changes now?